lib/mailchimp_api/resources/member.rb

```diff
@@ -10,6 +10,16 @@ class Member < Base
 
     self.prefix = '/3.0/lists/:list_id/'
 
+    # The path to get notes is '/3.0/lists/:list_id/members/:member_id'
+    # Unfortunately, ActiveResource does not support nested resources, only single parent resources:
+    #
+    # https://github.com/rails/activeresource/blob/4accda8bc03ceae0ad626f8cec0b4751e89a58ad/lib/active_resource/associations.rb#L151
+    #
+    # Using a has_many, there is no way to include the `list_id`, so we just create our own notes method.
+    def notes
```

Comment:
Should this support pagination?

Reply:
Hmm. Hadn't considered. Shouldn't all endpoints support pagination?

Reply:
I'm sure the endpoint does, but this method doesn't